programs/dynamic-bonding-curve/src/instructions/admin/ix_claim_pool_creation_fee.rs

```diff
@@ -0,0 +1,49 @@
+use anchor_lang::prelude::*;
+use crate::{constants::fee::TOKEN_2022_POOL_WITH_OUTPUT_FEE_COLLECTION_CREATION_FEE, state::*, *, EvtClaimPoolCreationFee};
+
+/// Accounts for withdraw creation fees
+#[event_cpi]
+#[derive(Accounts)]
+pub struct ClaimCreationFeeCtx<'info> {
+    #[account(mut)]
+    pub pool: AccountLoader<'info, VirtualPool>,
+
+    /// Claim fee operator
+    #[account(has_one = operator)]
+    pub claim_fee_operator: AccountLoader<'info, ClaimFeeOperator>,
+
+    /// Operator
+    pub operator: Signer<'info>,
+
+    /// CHECK: treasury
+    #[account(
+        mut, 
+        address = treasury::ID
+    )]
+    pub treasury: UncheckedAccount<'info>,
+
+    pub system_program: Program<'info, System>,
+}
+
+pub fn handle_claim_pool_creation_fee(ctx: Context<ClaimCreationFeeCtx>) -> Result<()> {
+    let pool = ctx.accounts.pool.load()?;
+
+    if pool.has_creation_fee() && !pool.creation_fee_claimed() {
```

Comment:
we can just calculate excessive lamport from pool account? and if excessive_lamport > 0, we transfer